scenarios/15b_Repelling_the_Orcs.cfg

```diff
@@ -85,6 +85,10 @@
 #ifdef HARD
             {OBJECTIVE_NOTES _"There are no hidden elements in this scenario. The Orcish leader can't be interrogated. This scenario is intended for gathering more experience for Gawen's troops. Remember, you will be forced to choose only twelve of your troops in the next scenario."}
 #endif
+#ifdef NIGHTMARE
+            #po: NIGHTMARE difficulty; previous three difficulties pretty much only vary on turn number specified, but here we also cut out a lot of the hints:
+            {OBJECTIVE_NOTES _"This is a grinding scenario. Remember, you'll be forced to choose only nine troops in the next scenario."}
```

Comment:
This is a grinding scenario actually kind of gives enough of a hint about what it is, so I'd remove that. In fact you could just remove the whole note on nightmare difficulty.

Reply:
Removed the whole note in [086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).